zazu/util.py

```diff
@@ -95,20 +95,30 @@ def prompt(text, default=None, expected_type=str):
     return expected_type(result)
 
 
-def pick(choices, message):
+def pick(choices, message, checkbox=False):
```

Comment:
I mistyped the reason to separate it is that there are 2 bits: allow_none and allow_multiple:

That leads to 4 possible combinations:
allow multiple and allow none (checkbox where you could pick nothing) <- only way to do this is  to catch Ctrl-C which i don't think we want to do...because generally that means exit.
allow multiple don't allow none (check box must pick at least 1)
dont' allow multiple allow none (list, add None as a possible choice)
don't allow multiple don't allow none (list)

So there are 3 valid combinations on the 2 bits, which leads me to make 2 functions one of which has the allow_none option.

    def pick(choices, message, allow_none=False):
        """Interactively pick from a list of possibilities."""
        if not choices:
            return None
        if allow_none:
            choices = [None] + choices
        if len(choices) > 1:
            click.clear()
            questions = [inquirer.List(' ', message=message, choices=choices)]
            response = inquirer.prompt(questions)
            if response is None:
                raise KeyboardInterrupt
            return response[' ']
        return choices[0]


    def pick_multiple(choices, message):
        """Interactively pick multiple items from a list of possibilities."""
        if not choices:
            return None
        click.clear()
        response = inquirer.prompt([inquirer.Checkbox(' ', message=message, choices=choices)])
        if response is None:
            raise KeyboardInterrupt
        return response[' ']